### PR TITLE
Review clouds-manual.md

### LINKED ISSUE #2798: Review clouds-manual.md

### PR DESCRIPTION
**Pull request by pmatulis**
resolves #2798

### CODE REVIEW

File: src/en/clouds-manual.md [hunk 1]
```diff
@@ -1,141 +1,212 @@
...
-deploy the new application.
+## Deploying a charm in a Manual cloud
 
-With a manual cloud, you need to create your machines in advance, so Juju will
-need to know which of the machines you wish to target when you deploy a charm
-(or scale out). This can be done with the use of the
-[placement directives][placement]: you need to use the
-'--to' option to specify the destination, which is either a machine or a
-container on a machine:
+To deploy WordPress onto the machine we added previously its ID (of '0') is
```
Comment (degville) [inline src/en/clouds-manual.md]: I'd maybe rewrite the 'its ID' part, or add a comma. eg. *, we need to declare the ID (of '0') of the machine.*

File: src/en/clouds.md [hunk 2]
```diff
@@ -176,10 +176,10 @@ session.
    
 ^# Manual
 
-   To add a Manual cloud, you need to supply a name you wish to call it, the
-   IP address (or hostname) used to connect to it, and what remote user account
-   to connect to (over SSH). This last is done by prepending 'user@' to the
-   address/hostname.
+   To add a Manual cloud, you need to supply a name you wish to call it, the IP
+   address (or hostname) for the machine you intend to use as a controller, and
+   what remote user account to connect to (over SSH). This last is done by
```
Comment (degville) [inline src/en/clouds.md]: maybe remove *last*.

File: src/en/clouds-manual.md [hunk 3]
```diff
@@ -1,141 +1,212 @@
...
 
-Juju caters for the
-case where you may not be able to access a more traditional cloud in a
-straightforward way; maybe you can't create additional instances, or perhaps
-your cloud is really a collection of disparate hardware.
+The purpose of the Manual cloud is to cater to the situation where you have
+machines (of any nature) at your disposal and you want to create a backing
+cloud out of them. If this collection of machines is composed solely of bare
+metal you might opt for a MAAS cloud but note that such machines would also
+require IPMI hardware and a MAAS infrastructure. The Manual cloud can therefore
```
[the remark this answers is not in the document]
Comment (pmatulis) [reply src/en/clouds-manual.md]: :white_check_mark:

File: src/en/clouds-manual.md [hunk 4]
```diff
@@ -1,141 +1,212 @@
...
+
+Your collection of machines (minus the controller machine) *must* be added to
+Juju by means of the `add-machine` command. A machine is specified by means of
+its IP address.
+
+!!! Important:
+    A Manual cloud requires at least one machine to be added.
+
+Finally, to deploy a charm the `deploy` command is used as normal. However, a
+machine *must* be targeted. This latter is accomplished with the `--to` option
```
[the remark this answers is not in the document]
Comment (pmatulis) [reply src/en/clouds-manual.md]: :white_check_mark:

File: src/en/clouds-manual.md (hunk 1 above)
Comment (pmatulis) [reply src/en/clouds-manual.md]: :heavy_check_mark:

File: src/en/clouds.md (hunk 2 above)
Comment (pmatulis) [reply src/en/clouds.md]: :white_check_mark:

**Pull request merged**